src/main/steps/applicant1/finalising-your-application/content.ts

```diff
@@ -26,9 +26,8 @@ const en = ({ isDivorce, partner, userCase, isJointApplication }: CommonContent)
     You may both have to come to a court hearing, if they apply.`,
     line3: 'If you wait a year before applying then you will need to explain the delay to the court.',
   },
+  readMoreJointSubheader: 'Changing to a sole application',
   readMoreJoint: {
-    subHeader: 'Changing to a sole application',
```

Comment:
you didnt need to remove this. could have been here. only the one below should have been in the correct object instead of here.

Reply:
Reverted that and added it to its correct object.